### DISCUSSION TITLE
Suggestions

### DISCUSSION THREAD

**Discussion by pranshu05** [pranshu05/Elpha#2] [Ideas]
Hello everyone

I made this discussion for suggestions

You can suggest me something to add in my bot.....Or you can reach me at discord : Pranshu05#4726

: )

**Comment by FadTheChad** [comment]
no

**Reply by pranshu05** [reply]
Good suggestion will hang it on the board

**Comment by tonyG433** [comment]
delete the bot

**Reply by pranshu05** [reply]
Never

**Reply by tonyG433** [reply]
why?

**Comment by FadTheChad** [comment]
- Change the README file to a original one

- Add command categories

- Unique commands

- Use Git or GitHub Desktop plez

**Reply by pranshu05** [reply]
Thanks for suggestion

Will add command catagories 

And i just installed Gitbash....will start using it

**Reply by pranshu05** [reply]
wdym by changing README?

**Reply by FadTheChad** [reply]
The README looks similar to an other bot and people will notice that, so rewrite it to a unique one
